server/routes/vocalists-router.js

```diff
@@ -0,0 +1,55 @@
+const { Router } = require('express')
+const uuid = require('uuid/v4')
+
+module.exports = function vocalistsRouter(vocalists) {
```

Comment:
I see you defined a `vocalistsRouter` here, but apparently the `producersRouter` is working with a different collection? 😄 Maybe just rename `producersRouter` to `artistsRouter`. It depends if you foresee defining different routes for producers and artists. In either case, be careful not to add code that your features don't use yet.